scripts/new_python_based_image.py

```diff
@@ -1,22 +1,24 @@
 import argparse
+import contextlib
+import logging
 import os
-import shutil
-import sys
 import platform
-import subprocess
-import logging
 import re
+import shutil
+import subprocess
+import sys
+from dataclasses import dataclass
 
 
 LOGGER = logging.getLogger(__name__)
 
 
-def configure_logger(log_level):
+def configure_logger(log_level: str):
```

Comment:
```suggestion
def configure_logger(log_level: str) -> None:
```

Reply:
That seems redundant to me but why not?!

Reply:
You'd get a warning when you actually attempt to return non None value, or when you try to use the return value from calling this. Guess that's pretty much it.